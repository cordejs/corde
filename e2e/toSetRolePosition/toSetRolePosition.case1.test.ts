import Utils from "../testUtils";
import { runCLI } from "../cliRunner";

it("should set role as mentionable", async () => {
  const command = Utils.buildCommandWithConfigPath("toSetRolePosition", "bot_case1.spec.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(stdout).toMatchSnapshot();
  expect(mockProcess).toBeCalledWith(0);
});
