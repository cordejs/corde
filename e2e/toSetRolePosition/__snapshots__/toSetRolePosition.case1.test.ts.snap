// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should set role as mentionable 1`] = `
"
 PASS   e2e\\\\toSetRolePosition\\\\__cordeTest__\\\\bot_case1.spec.ts   100ms
   âœ”  should increase a role position   100ms

Test Files: 1 passed. 1 total
Tests:      1 passed. 1 total
Time:       100ms
"
`;
